CONTRIBUTING.rst

```diff
@@ -1,29 +1,24 @@
 Developing
 ==========
 
-Install dependencies:
+This project uses `Tox <https://tox.readthedocs.io/en/latest/config.html>`_ to manage virtual environments.
 
-::
+To run the tests::
 
-    pip install treq[dev]
+    tox -e py38-twisted_latest
```

Comment:
just a minor comment.... feel free to ignore as I might be wrong :)

It would be nice to name the tox test envs like `py38-test-twisted_latest``. 
In this way,  you can call `tox -e test-twisted_latest` and it will run with your default python.

Reply:
Actually, you can do `tox -e twisted_latest` to run with your default Python as-is. The default commands are to test.